plugins/cli/README.md

```diff
@@ -12,20 +12,22 @@ plugin.conf
 #   ip = 127.0.0.1
 #   port = 2323
 #   update = false
+   hashed_password = 1245a9633edf47b7091f37c4d294b5be5a9936c81c5359b16d1c4833729965663f1943ef240959c53803fedef7ac19bd59c66ad7e7092d7dbf155ce45884607d
 </pre>
 
 This plugin listens for a telnet connection. 
 <code>ip = </code> used network interface, e.g. 127.0.0.1 (localhost, default) or listen on all network interfaces: 0.0.0.0
 <code>port =</code> used network port, default 2323
 <code>update =</code> restrict the access of the items to read only (false, default) or allows read/write access (true)
-
+<code>hashed_password = </code> pasword that needs to be entered on login. SHA-512 hashed. Value shown above is "very_secure_password"
```

Comment:
Typo in "password". Do we want to use sha512password instead of hashed_password? In #101 we discussed the hashed password support a little bit and there it is named md5password (so one see which hash algorithm is used). I think we should be consistent in both plugins.